cli/command_line_options.go

```diff
@@ -11,18 +11,19 @@ import (
 
 // CommandLineOptions stores information that was requested by the user from the CLI.
 type CommandLineOptions struct {
-	Body           string
-	Headers        map[string][]string
-	FollowLocation bool
-	FileToUpload   string
-	MaxRedirect    int
-	Method         string
-	OutputFile     string
-	Profiles       []string
-	RequestName    string
-	URL            string
-	Values         map[string][]string
-	Variables      map[string]string
+	Body            string
+	Headers         map[string][]string
+	FollowLocation  bool
+	FileToUpload    string
+	MaxRedirect     int
+	Method          string
+	OutputFile      string
+	PostProcessFile string
```

Comment:
Minor nit: `PostProcessFile` really represents a file name (maybe even a file path?) and not a `os.File` `struct`, for example, (or a file's contents that have been converted to a `string`).

Obviously, the type helps clarify this, I'm just being nitpicky 😅

Reply:
Since I have to marshall these things to send them to the daemon, I prefer to keep them as strings and rely on the variable name to determine the type.
This will probably change after the refactor because I'm also planning to support raw JavaScript from profiles. Which means I'll just load files before sending to the daemon and have this be a `PostProcessScript` variable.